libraries/core_libs/consensus/include/transaction_manager/transaction_queue.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "transaction/transaction.hpp"
+
+namespace taraxa {
+
+class TransactionQueue {
```

Comment:
If you use doxygen comments for class description and some important methods, people would not need much much less time to uderstand how exactly this pool works (all the comparing based on nonce, fees, etc...) and it take 5 minutes of your time. Would be nice to comment each class and method, which is not just some dummy data holder.... Definitely saves time => money if someone needs to touch it in the future